www/_posts/2019-04-11-nodejs-6.x-8.x-deprecation-timeline.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title: "Node 6.x and 8.x Deprecation Timeline for Apache Cordova"
+categories: news
+tags: deprecation timeline node
+---
+
+Currently, all of the latest released platforms, tools, and core libraries require a minimum version of `node 6.x` to be installed. Additionally, we have added support for `node 10.x`.
+
+The Node.js Foundation has reported that the End-of-Life (EOL) for version 6.x will be April 30th, 2019. Looking ahead, the Node.JS Foundation has also scheduled the deprecation of 8.x on December 31, 2019 to be aligned with the scheduled End-of-Life of OpenSSL-1.0.2.
+
+Since October 30th, 2018, `node 10.x` has started its active Long Term Support ([LTS][1]) period. Thus, as `node 10.x` is the default download from [Node.js](https://nodejs.org/en/), we also recommend users to upgrade to the current LTS (10.x) before/by the time 8.x has reached EOL.
+
+During these two distinct deprecation periods, we will prepare our tools, platforms, core libraries, and plugins to follow the Node.js Foundation reported EOL schedule.
+
+We will also plan to add `node 12.x` support. Currently, `node 12.x` is pending for release on April 23rd, 2019.
+
+JavaScript files in plugins themselves are unaffected since the JavaScript support for them is dependent on platform browser support. For the Electron platform, if using Node.js APIs there is a possibility of experience an effect.  
+
+<!--more-->
+
+## Cordova Timeline
+
+### **March 2019**
+
+* All AppVeyor/Travis CI builds will continue to test on Node.js 6.x, 8.x, and 10.x.
+
+### **April 30th, 2019**
+
+* All AppVeyor/Travis CI builds will test on Node.js 6.x, 8.x, 10.x, and 12.x.
+* Update requirement check and notify users that using Node.js 6.x, was deprecated by the Node team and is not recommended to use and should upgrade to the minimum LTS (10.x).
+
+### **September 2019**
+
+* Update requirement check and notify deprecation warning if the user is using Node.js 6.x and 8.x and recommend to upgrade to the minimum LTS (10.x).
+
+### **December 31st, 2019**
+
+* Deprecate use of Node.js 6.x
```

Comment:
```suggestion
* Deprecated use of Node.js 6.x
```

Reply:
We should deprecate now, and remove support at the end of the year, or sooner for 6.